Documentation/debugging/unix-instructions.md

```diff
@@ -6,10 +6,40 @@ CoreFX can be debugged on unix using both lldb and visual studio code
 ## Using lldb and SOS
 
 - Run the test using msbuild at least once with `/t:BuildAndTest`.
-- Install version 3.6 of lldb and launch lldb with corerun as the process and arguments matching the arguments used when running the test through msbuild.
+- Install version 3.6 of lldb and launch lldb with dotnet as the process and arguments matching the arguments used when running the test through msbuild.
```

Comment:
I'm not sure. I think the 3.8 requirement may be specific to core dumps. @janvorli Do you know what version is required for different things? Is lldb 3.8 required for any type of debugging?